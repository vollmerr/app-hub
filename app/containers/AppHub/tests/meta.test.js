import appHub, { meta } from '../meta';

describe('meta', () => {
  it('should be in the correct format', () => {
    expect(meta).toMatchSnapshot();
    expect(appHub).toMatchSnapshot();
  });
});
