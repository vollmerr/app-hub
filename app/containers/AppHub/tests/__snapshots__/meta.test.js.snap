// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`meta should be in the correct format 1`] = `
Object {
  "apphub": Object {
    "desc": "Central location for CDT ODI applications",
    "keywords": "App, Hub",
    "title": "App hub",
  },
  "bars": Object {
    "desc": "Badge Access Request System",
    "keywords": "badge, employee, new, student, request",
    "title": "BARS",
  },
  "cold": Object {
    "desc": "",
    "keywords": "",
    "title": "COLD",
  },
  "ed": Object {
    "desc": "Employee Directory",
    "keywords": "Employee, Directory, Lookup",
    "title": "ED",
  },
  "loading": Object {
    "desc": "test...",
    "keywords": "test, loading",
    "title": "TEST LOADING",
  },
  "spa": Object {
    "desc": "Document Acknowledgment",
    "keywords": "SPA, Approval, Acknowlegde, Document",
    "title": "SPA",
  },
}
`;

exports[`meta should be in the correct format 2`] = `
Object {
  "desc": "Central location for CDT ODI applications",
  "keywords": "App, Hub",
  "title": "App hub",
}
`;
